src/pages/ReimbursementAccount/IdentityForm.js

```diff
@@ -132,13 +139,63 @@ const IdentityForm = ({
+                        containerStyles={[styles.mt4]}
+                        value={getFormattedAddressValue()}
+                        onChangeText={(fieldName, value) => onFieldChange(fieldName, value)}
+                        errorText={errors.street ? translate('bankAccount.error.addressStreet') : ''}
+                    />
+                    <TextLink
+                        style={[styles.textMicro]}
+                        onPress={() => onFieldChange('manualAddress', true)}
+                    >
+                        Can&apos;t find your address? Enter it manually
+                    </TextLink>
+                </div>
+            )}
+            {manualAddress && (
+                <div>
+                    <ExpensiTextInput
+                        label={translate('common.personalAddress')}
+                        containerStyles={[styles.mt4]}
+                        value={street}
+                        onChangeText={value => onFieldChange('addressStreet', value)}
+                        errorText={errors.street ? translate('bankAccount.error.address') : ''}
+                    />
+                    <Text style={[styles.mutedTextLabel, styles.mt1]}>{translate('common.noPO')}</Text>
+                    <View style={[styles.flexRow, styles.mt4]}>
+                        <View style={[styles.flex2, styles.mr2]}>
+                            <ExpensiTextInput
+                                label={translate('common.city')}
+                                value={city}
+                                onChangeText={value => onFieldChange('addressCity', value)}
+                                errorText={errors.city ? translate('bankAccount.error.addressCity') : ''}
+                                translateX={-14}
+                            />
+                        </View>
+                        <View style={[styles.flex1]}>
+                            <StatePicker
+                                value={state}
+                                onChange={value => onFieldChange('addressState', value)}
+                                errorText={errors.state ? translate('bankAccount.error.addressState') : ''}
+                                hasError={Boolean(errors.state)}
+                            />
+                        </View>
+                    </View>
+                    <ExpensiTextInput
+                        label={translate('common.zip')}
+                        containerStyles={[styles.mt4]}
+                        keyboardType={CONST.KEYBOARD_TYPE.NUMERIC}
+                        value={zipCode}
+                        onChangeText={value => onFieldChange('addressZipCode', value)}
+                        errorText={errors.zipCode ? translate('bankAccount.error.zipCode') : ''}
+                        maxLength={CONST.BANK_ACCOUNT.MAX_LENGTH.ZIP_CODE}
+                    />
+                </div>
+            )}
```

Comment:
I would also use a ternary here for brevity but it's honestly NAB if you prefer this.

```suggestion
            {!manualAddress ? (
                <div>
                    <AddressSearch
                        label={translate('common.personalAddress')}
                        containerStyles={[styles.mt4]}
                        value={getFormattedAddressValue()}
                        onChangeText={(fieldName, value) => onFieldChange(fieldName, value)}
                        errorText={errors.street ? translate('bankAccount.error.addressStreet') : ''}
                    />
                    <TextLink
                        style={[styles.textMicro]}
                        onPress={() => onFieldChange('manualAddress', true)}
                    >
                        Can&apos;t find your address? Enter it manually
                    </TextLink>
                </div>
            ) : (
                <div>
                    <ExpensiTextInput
                        label={translate('common.personalAddress')}
                        containerStyles={[styles.mt4]}
                        value={street}
                        onChangeText={value => onFieldChange('addressStreet', value)}
                        errorText={errors.street ? translate('bankAccount.error.address') : ''}
                    />
                    <Text style={[styles.mutedTextLabel, styles.mt1]}>{translate('common.noPO')}</Text>
                    <View style={[styles.flexRow, styles.mt4]}>
                        <View style={[styles.flex2, styles.mr2]}>
                            <ExpensiTextInput
                                label={translate('common.city')}
                                value={city}
                                onChangeText={value => onFieldChange('addressCity', value)}
                                errorText={errors.city ? translate('bankAccount.error.addressCity') : ''}
                                translateX={-14}
                            />
                        </View>
                        <View style={[styles.flex1]}>
                            <StatePicker
                                value={state}
                                onChange={value => onFieldChange('addressState', value)}
                                errorText={errors.state ? translate('bankAccount.error.addressState') : ''}
                                hasError={Boolean(errors.state)}
                            />
                        </View>
                    </View>
                    <ExpensiTextInput
                        label={translate('common.zip')}
                        containerStyles={[styles.mt4]}
                        keyboardType={CONST.KEYBOARD_TYPE.NUMERIC}
                        value={zipCode}
                        onChangeText={value => onFieldChange('addressZipCode', value)}
                        errorText={errors.zipCode ? translate('bankAccount.error.zipCode') : ''}
                        maxLength={CONST.BANK_ACCOUNT.MAX_LENGTH.ZIP_CODE}
                    />
                </div>
            )}
```

Reply:
Same as above :)